client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -63,10 +70,27 @@ const useUsersManagement = ({ page, rowsPerPage}: useUsersManagementInCome) : us
         fetchUsers();
     }, [page])
     
+    const watchUserInfo = (row: any) => {
```

Comment:
Why  do you convert the object every time you open the dialog? Shouldn’t you just do it once you load them all?